gazebosim/gz-sim
# Adds a tool for environment data visualization

# 🎉 New feature

Depends on:
* https://github.com/gazebosim/gz-math/pull/508
* https://github.com/gazebosim/gz-common/pull/446
* https://github.com/gazebosim/gz-gui/pull/494
~~* https://github.com/gazebosim/gz-sim/pull/1691~~

## Summary
This PR adds a tool to visualize Scalar Environmental Data. 

![pointcloud_viz](https://user-images.githubusercontent.com/542272/194438423-1d3bebc0-8de3-4408-bd8d-bb4b80470b4a.gif)

## Test it
To use it is currently a bit cumbersome, but the following steps apply. I'll probably add a tutorial for this soon:
1. Open the `Environment Loader` plugin. Load the environment in.
2. Open the `Environment Visuallization panel to enable visuallization`
3. Load the `Pointcloud Plugin` to see the actual data.

## Checklist

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.
